doc/rst/meta/modules/standard.rst

```diff
@@ -6,19 +6,111 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+
+Automatic Modules
+-----------------
+
 All Chapel programs automatically ``use`` the modules :chpl:mod:`Builtins` ,
 :chpl:mod:`Math` , and :chpl:mod:`Types` by default.
 
+.. toctree::
+   :maxdepth: 1
+
+   Builtins <standard/Builtins>
+   Math <standard/Math>
+   Types <standard/Types>
+
+
+Data Structures
+---------------
+
+.. toctree::
+   :maxdepth: 1
+
+   Heap <standard/Heap>
+   List <standard/List>
+   Map <standard/Map>
+   Set <standard/Set>
+
+   
+Diagnostics
+-----------
+
+.. toctree::
+   :maxdepth: 1
+
+   CommDiagnostics <standard/CommDiagnostics>
+   Memory <standard/Memory>
+   Time <standard/Time>
```

Comment:
Somehow `Time` does not sound like diagnostics to me. I'd move it under Utilities.

Or, we can have it in both categories.

Reply:
I don't think we should have any module in more than one category.

`Time` is also open for discussion and can change or the module might disappear altogether as part of https://github.com/chapel-lang/chapel/issues/16394. But regardless, when you think of it, "Diagnostics" are actually "Utilities", so should we do it hierarch....  nevermind :)

Reply:
Just pointing out that with the automatic modules, we already have modules in multiple categories (and I think it is reasonable to do so)

Reply:
Time's almost certain to go away, so I don't know that we should obsess about it too much...

Reply:
I moved it under Utilities for now.